python/cuml/test/dask/test_pca.py

```diff
@@ -70,6 +71,56 @@ def test_pca_fit(nrows, ncols, n_parts, input_type, client):
         assert array_equal(cuml_res, skl_res, 1e-1, with_sign=with_sign)
 
 
+@pytest.mark.mg
+@pytest.mark.parametrize("nrows", [1000])
+@pytest.mark.parametrize("ncols", [20])
+@pytest.mark.parametrize("n_parts", [5])
+@pytest.mark.parametrize("input_type", ["dataframe", "array"])
+def test_pca_tsqr(nrows, ncols, n_parts, input_type, ucx_client):
+
+    from cuml.dask.decomposition import PCA as daskPCA
+    from sklearn.decomposition import PCA
+
+    from cuml.dask.datasets import make_blobs
+
+    X, _ = make_blobs(n_samples=nrows,
+                        n_features=ncols,
+                        centers=1,
+                        n_parts=n_parts,
+                        cluster_std=0.5,
+                        random_state=10, dtype=np.float32)
+
+    wait(X)
```

Comment:
Does the impact the test?

Reply:
Not really. Can take it out. Not sure why it was there in the first place